### PR TITLE
es6-operators -- Tim Woo-Sam

### PR DESCRIPTION
**Pull request by twoosam**
image

### CODE REVIEW

File: es6-operators/main.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+console.log(v1 && 'v1 is truthy');
+console.log(v2 && 'v2 is truthy');
```
Comment (shawnkost) [inline es6-operators/main.js:14]: This is incorrect. The way this is structured, you are console logging regardless if the value is truthy or not. The console log should only ever run if the value is truthy.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Tim!

**Comment by twoosam** [comment]
image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**